OpenRA.Mods.Common/Traits/Attack/AttackFollow.cs

```diff
@@ -93,6 +93,10 @@ public override Activity Tick(Actor self)
 				if (self.IsDisabled())
 					return this;
 
+				var mobile = move as Mobile;
+				if (mobile != null && !mobile.CanInteractWithGroundLayer(self))
+					return this;
```

Comment:
No, I'm not sure about this. I'm not sure how to make the unit repath instead either, though.